docs/api/rest-endpoints.md

```diff
@@ -1,12 +1,488 @@
 # Fleet REST API endpoints
 
+## Authentication
+
+Making authenticated requests to the Fleet server requires that you are granted permission to access data. The Fleet Authentication API enables you to receive an authorization token. The typical steps to making an authenticated API request is outlined below.
```

Comment:
```suggestion
Making authenticated requests to the Fleet server requires that you are granted permission to access data. The Fleet Authentication API enables you to receive an authorization token. The typical steps to making an authenticated API request are outlined below.
```

Reply:
Also let's add a paragraph with something like "All Fleet API requests are authenticated unless noted in the documentation." Then I think we can remove the "header" sections in each individual endpoint.

Reply:
Sounds good. I like that clarification.